src/config/example.toml

```diff
@@ -146,3 +146,4 @@ bloom_false_positive = 0.001
 backup_storage_url = "memory"
 backup_storage_directory = "backup"
 telemetry_enabled = true
+max_concurrent_creating_streaming_jobs = 1
```

Comment:
it might be more suitable to be placed under `[meta]`?

Reply:
I placed it under `[system]` so that it can be modified at runtime without restarting meta.